courses/free-courses-ar.md

```diff
@@ -462,4 +462,9 @@
 * [Arabic Angular and Firebase App تطبيق انجولار وفايربيز](https://www.youtube.com/playlist?list=PLL2zWZTDFZzh2WEmc3fH_O4y4N05ZCqB2) - KMR Script&rlm; (YouTube)
 * [Arabic NgRx (Angular + Redux)&rlm;](https://www.youtube.com/playlist?list=PLL2zWZTDFZzhW10baUv1esvrowMwbfd5H) - KMR Script&rlm; (YouTube)
 
+
+### iOS
+* [iOS & Xcode دورة برمجة تطبيقات الايفون باستخدام لغة سويفت  ](https://www.youtube.com/playlist?list=PLQaOY10EEc8bNbEBMyiJU1I-GIgs1LQfj) - بامبرمج
```

Comment:
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.